datafusion/core/src/bin/print_functions_docs.rs

```diff
@@ -48,12 +49,13 @@ fn main() {
         _ => {
             panic!("Unknown function type: {}", function_type)
         }
-    };
+    }?;
 
     println!("{docs}");
```

Comment:
do we need this println?

Reply:
Yes. This is how the shell script that builds the .md files retrieves the text. see https://github.com/apache/datafusion/blob/main/dev/update_function_docs.sh for impl.